### ISSUE TITLE
366

### ISSUE THREAD

**Issue by davidamichelson** [srophe/srophe-app-data#1081]

**Comment by davidamichelson** [comment]
Needs a URI for Addai Scher, “Analyse de l’histoire de Rabban Bar ‘Edta, Moine Nestorien Du VIe Siècle,” Revue de l’Orient chrétien II, 1:4 [11] (1906): 403–423; 2:1 [12] (1907), 9–13.

**Comment by davidamichelson** [comment]
@wlpotter So in this case, CBSS has one entry and we need to split it into two:
https://www.zotero.org/groups/4861694/items/F8575U6V

Should I wait for now? See http://syriaca.org/person/366 bibl xml:id="bib366-7"

**Comment by wlpotter** [comment]
I think this is worth discussing. So this is a two-part article published in separate issues of the same journal, right? I've seen other cases in CBSS, as well as related issues like whether to treat the edition and translation of a PO or CSCO volume as two records or one. Also related to part-to-whole issues.

All that to say, yes, let's wait on this and discuss what should be our normal way of handling

**Comment by wlpotter** [comment]
I suppose this also raises the question of if this becomes two bibl citations in the person record, one for each part of the article?
